apps/src/code-studio/components/Grid.jsx

```diff
@@ -182,12 +182,12 @@ var Grid = React.createClass({
       return;
     }
 
-    var top = Math.min(dragStart.row, row),
-        left = Math.min(dragStart.col, col),
-        bottom = Math.max(dragStart.row, row),
-        right = Math.max(dragStart.col, col);
+    const top = Math.min(dragStart.row, row),
+          left = Math.min(dragStart.col, col),
+          bottom = Math.max(dragStart.row, row),
+          right = Math.max(dragStart.col, col);
```

Comment:
nit: isnt this too much indenting? should be either 2 or 4 spaces

Reply:
:+1: Was following the original pattern here, but happy to update.

Reply:
I dont *really* care too much either way, but one could argue that the original pattern is okay because it's 5 spaces, and the latter pattern is not because it's five (though both align with the text from the first line).